en/docs/debug.md

```diff
@@ -46,20 +44,20 @@ Follow the steps below to start a debug session with configurations. All the con
 
 2. Press the **Control + Shift + D** keys (for Mac: **Command + Shift +D**) to launch the Debugger view.
 
-3. Click **create a launch.json** file and then select **Ballerina Debug** as the **Environment**.
+3. Click **create a launch.json** file and then select **Ballerina Debug** as the **Environment**. 
 
-   You can view the opened `launch.json` file.
+      You view the opened `launch.json` file.
```

Comment:
I think `You can` is more clear.
Or better - `You will see the...`

but not an error.

Reply:
isn't the best practice to use direct voice in technical documentation?

Reply:
We shouldn't be using `will`, yes. Need to change that in multiple places.

`You view.. ` in this sentence seems very odd IMO. But let's proceed with your text.

Reply:
Yes, I agreed to @nilminiwso2's point here. `You view the ...` seems incomplete or odd.

Reply:
Changed to `You will see the...`.